package.json

```diff
@@ -7,9 +7,10 @@
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
     "@types/styled-components": "^5.1.26",
+    "dotenv": "^16.0.3",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
-    "react-scripts": "5.0.1",
+    "react-scripts": "^5.0.1",
```

Comment:
I have vulnerability, then run `npm install`
<img width="641" alt="Screenshot 2022-11-21 at 10 02 31" src="https://user-images.githubusercontent.com/14818162/202985746-ba8dc1b8-29e0-487f-b5bd-3f956f7c33d3.png">

Reply:
The article written by CRA creator states that this vulnerability can be ignored
https://overreacted.io/npm-audit-broken-by-design/